source/development/packaging/generate-ova.rst

```diff
@@ -3,60 +3,66 @@
 .. meta::
   :description: Wazuh provides an automated way of generating a Virtual machine in OVA format. Learn how to build a Virtual machine with Wazuh central components installed in this section.  
 
-.. _create-ova:
-
 Virtual machine
 ===============
 
-Wazuh provides an automated way of generating a Virtual machine in OVA format that is ready to run a Wazuh manager and ELK.
-
-To create the virtual machine follow these steps:
+We provide an automated way of generating a virtual machine (VM). The ``generate_ova.sh`` script creates a VM in OVA format. The generated VM is ready to run the Wazuh central components.
 
 Requirements
-^^^^^^^^^^^^
+------------
+
+-  `Virtual Box <https://www.virtualbox.org/manual/UserManual.html#installation>`__
+-  `Vagrant <https://www.vagrantup.com/docs/installation/>`__
+-  `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+-  `Python <https://www.python.org/download/releases/2.7/>`__
 
- * `Virtual Box <https://www.virtualbox.org/manual/UserManual.html#installation>`_
- * `Vagrant <https://www.vagrantup.com/docs/installation/>`_
- * `Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
- * `Python <https://www.python.org/download/releases/2.7/>`_
+We recommend using a system with at least the following specifications.
 
-Download our wazuh-packages repository from GitHub and go to the ova directory.
+-  8 GB RAM
```

Comment:
For consistency with other sections of the documentation, consider using a table to specify the hardware requirements. 

<img src="https://github.com/wazuh/wazuh-documentation/assets/61882981/aa7608a6-82a9-4867-a506-74ad76ebeecc"  width="500" />